doc/api/assert.md

```diff
@@ -163,6 +163,10 @@ An alias of [`assert.ok()`][].
 <!-- YAML
 added: v0.1.21
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: The type tags are now properly compared and a couple minor
```

Comment:
Nit: Perhaps change the wording to "... and *there are* a couple *of* minor ..."